CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
  * Added Consumer `SeekPartitions()` method to seek multiple partitions at
    once and deprecated `Seek()`.
+ * Added Integration-Testsuite which runs all the integration tests in one go, to enable per PR builds and Integration Tests
```

Comment:
The changelog should have changes that the end-user cares about - that includes additions/updates to the API, or bug fixes, or behavior changes.
This shouldn't be included as it doesn't affect the end user directly

Reply:
Gotcha!